ghost/core/test/unit/server/models/post.test.js

```
@@ -2,6 +2,7 @@
 const errors = require('@tryghost/errors');
 const should = require('should');
 const sinon = require('sinon');
+const { Parser } = require('node-sql-parser');
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)